azure-pipelines.nightly.yml

```diff
@@ -59,12 +59,6 @@ parameters:
       DeviceType: Desktop
       DeviceName:
       TestCaseFilter: TestCategory=Smoketest-Extended
-    - OS: Windows
```

Comment:
can you do this to the beta tests too please

Reply:
Beta tests are fine. Edge is not included, only EdgeChromium.